saleor/product/templatetags/product_tags.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+from ..utils import get_pricing_info
+
+register = template.Library()
+
+
+@register.assignment_tag
+def pricing_info_for_product(product, discounts=None, local_currency=None):
+    return get_pricing_info(product, discounts, local_currency)
```

Comment:
After all of the experience with Oscar seeing this available at the template level makes me scared. We should be moving logic out of the template if anything.